src/context.rs

```diff
@@ -31,6 +33,8 @@ pub struct RectangleVec {
 }
 
 impl Drop for RectangleVec {
+    /// Unconditionally frees rectangle_list and all associated references. After this call,
+    /// the rectangle_list pointer must not be dereferenced.
     fn drop(&mut self) {
```

Comment:
I was wondering when documenting but you're right, with a step backward, it doesn't make sense.